.github/workflows/py-formatting.yml

```diff
@@ -14,5 +14,7 @@ jobs:
       - uses: actions/setup-python@v4
         with:
           python-version: "3.11"
+      - name: disable pre-commit validation
+        run: echo "SKIP_DATASET_CONFIG_VALIDATION=true" >> $GITHUB_ENV
```

Comment:
Curious on if we can simplify using the `SKIP` env variable. https://pre-commit.com/#temporarily-disabling-hooks 

That way, we don't need the additional check in the `.pre-commit-config.yaml`. :)

Reply:
Done!